app/views/quotes/_quote.html.erb

```diff
@@ -1,23 +1,35 @@
 <li class="py-4">
   <div class="flex space-x-3 text-sm">
-    <%= link_to quote.user, class: "self-center" do %>
-      <% gravatar_for quote.user, class: "h-8 w-8 rounded-full" %>
+    <% if quote.anonymous? %>
+      <div class="self-center">
+        <%= image_tag 'anonymous.svg', class: "w-7 h-7 rounded-full" %>
+      </div>
+    <% else %>
+      <%= link_to quote.user, class: "self-center" do %>
+        <% gravatar_for quote.user, class: "h-8 w-8 rounded-full" %>
+      <% end %>
     <% end %>
     <div class="flex-col flex-1">
       <div class="flex flex-col sm:flex-row justify-between self-center">
         <h3 class="flex-grow font-medium"><%= quote.text %></h3>
         <span class="relative z-0 inline-flex rounded-md">
-          <%= link_to edit_quote_path(quote), class: 'inline-flex items-center p-1 border border-gray-300 text-xs font-medium rounded-l-md text-gray-900 bg-white hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-hamers-red-500' do %>
-            <%= heroicon('pencil', class_name: "h-4 w-4") %> <span class="sm:hidden">Wijzigen</span>
+          <%= unless quote.anonymous?
```

Comment:
Nu krijgt iedereen voor iedere quote deze optie te zien, dat is minder logisch. Misschien een methode `can_be_anonymized_by?(user)` in de `Quote` model waar je het voor de specifieke user checkt. 

Dan kun je hier `if quote.can_be_anonymized_by?(current_user)` van maken.